coreos-base/coreos-metadata/files/coreos-metadata.target

```diff
@@ -0,0 +1,3 @@
+[Unit]
```

Comment:
To decouple from the coreos-metadata implementaiton you can do a couple things: 

 - rename to just metadata.target, perhaps install via the init repo instead.
 - use `systemd_enable metadata.target coreos-metadata.service` to add the .wants link

Effectively the same but makes more of a logical split which will be help if we do venture into the realm of alternate metadata sources.